src/legacy/core_plugins/interpreter/public/functions/font.ts

```diff
@@ -82,45 +82,43 @@ export function font(): ExpressionFunction<'font', null, Arguments, Style> {
       italic: {
         default: false,
         help: i18n.translate('interpreter.functions.font.args.italicHelpText', {
-          defaultMessage: 'Italicize, true or false',
+          defaultMessage: 'Italicize the text?',
         }),
         options: [true, false],
         types: ['boolean'],
       },
       lHeight: {
         aliases: ['lineHeight'],
         help: i18n.translate('interpreter.functions.font.args.lHeightHelpText', {
-          defaultMessage: 'Line height ({px})',
-          values: {
-            px: 'px',
-          },
+          defaultMessage: 'The line height in pixels',
         }),
         types: ['number', 'null'],
       },
       size: {
         default: 14,
         help: i18n.translate('interpreter.functions.font.args.sizeHelpText', {
-          defaultMessage: 'Font size ({px})',
-          values: {
-            px: 'px',
-          },
+          defaultMessage: 'The font size in pixels',
         }),
         types: ['number'],
       },
       underline: {
         default: false,
         help: i18n.translate('interpreter.functions.font.args.underlineHelpText', {
-          defaultMessage: 'Underline the text, true or false',
+          defaultMessage: 'Underline the text?',
         }),
         options: [true, false],
         types: ['boolean'],
       },
       weight: {
         default: 'normal',
         help: i18n.translate('interpreter.functions.font.args.weightHelpText', {
-          defaultMessage: 'Set the font weight, e.g. {examples}',
+          defaultMessage: 'The font weight. For example, {list}, or {end}.',
           values: {
-            examples: Object.values(FontWeight).join(', '),
+            list: Object.values(FontWeight)
+              .slice(0, -1)
+              .map(weight => `\`"${weight}"\``)
```

Comment:
nit: I would argue that `"` *and* code ticks are a bit redundant.

Reply:
As a formatting decision, I chose to wrap any argument values in back ticks so this show up as inline code. Again related to other comment about markdown, it will render correctly in the docs when they're published, but they will look kinda funky in autocomplete until we can add support for markdown syntax, which I'm hoping I can tackle in the next release (I'm assuming it's a counts as a feature).